src/interfaces/form-interface.ts

```diff
@@ -2,5 +2,6 @@ import Vue from 'vue'
 
 export interface FormIF extends Vue {
   reset(): void;
+  resetValidation(): void;
```

Comment:
Ref: https://vuetifyjs.com/en/api/v-form/#functions

(Also, my new component uses this.)